src/Framework/TestResult.php

```diff
@@ -244,6 +254,29 @@ public function addError(PHPUnit_Framework_Test $test, Exception $e, $time)
      * The passed in exception caused the failure.
      *
      * @param PHPUnit_Framework_Test                 $test
+     * @param PHPUnit_Framework_AssertionWarning $e
```

Comment:
I think you forgot to update the copied dockblock, here, the current content is:

> Adds a failure to the list of failures.
> The passed in exception caused the failure.

I think you can just replace _failure_ by _warning_.